tests/phpunit/includes/MarkupComparison.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests;
+
+/**
+ * Helper trait for comparing strings of HTML markup.
+ *
+ * @package AmpProject\AmpWP
+ */
+trait MarkupComparison {
```

Comment:
Note this trait was just updated to fix a bug: https://github.com/ampproject/amp-wp/pull/5421